hangman.js

```diff
@@ -1 +1,97 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var head = "___________\n |/       |\n |       (_)\n |\n |\n |\n |\n |____";
+var neck = "___________\n |/       |\n |       (_)\n |        |\n |\n |\n |\n |____";
+var body = "___________\n |/       |\n |       (_)\n |        |\n |        |\n |\n |\n |____";
+var arm1 = "___________\n |/       |\n |       (_)\n |       \\|\n |        |\n |\n |\n |____";
+var arm2 = "___________\n |/       |\n |       (_)\n |       \\|/\n |        |\n |\n |\n |____";
+var leg1 = "___________\n |/       |\n |       (_)\n |       \\|/\n |        |\n |       /\n |\n |____";
+var leg2 = "___________\n |/       |\n |       (_)\n |       \\|/\n |        |\n |       / \\\n |\n |____";
+
+var game = {
+  lettersInWord: [],
+  misses: [],
+  correct: [],
+  man: "",
+  parts: [head, neck, body, arm1, arm2, leg1, leg2],
+
+  showMenu: function() {
+    var option = prompt("Welcome to Hangman!\n\n[1] Play Game\n[2] How To Play\n[3] Quit");
+    if (option == 1) {
+      this.newGame();
+    } else if (option == 2) {
+      alert("Hangman is a guessing game where one player thinks of a word and a second player attempts to guess that word in seven (or less) guesses buy suggesting letters.\n\nPlayer 1 will enter a word into the appropriate prompt and the remaining player will enter guesses one letter at a time.\n\nThe game will end when either Player 2 correctly guesses the word or 'The Hangman' is drawn completely.");
+      this.showMenu();
+    }
+  },
+
+  getANewWord: function() {
+    var word = prompt("Player 1: Give a word to guess:");
+    this.lettersInWord = word.split("");
+  },
+
+  createCorrectBlanks: function() {
+    var self = this;
+    this.lettersInWord.forEach(function () {
```

Comment:
doesnt exactly seem like a necessary use for a `forEach` loop. Does the job, but a `for x in y` might be a more semantic implementation.